dashboard/src/pages/packages/index.vue

```diff
@@ -120,9 +120,8 @@ const toggleLegend = () => {
               }"
               aria-label="Go to first page"
               @click.prevent="packageStore.fetchPackages(1)"
-            >
-              <IconSkipStartFill /> First
-            </a>
+              ><IconSkipStartFill
```

Comment:
But use the title attribute, to provide a tooltip, just in case.

Reply:
Done.